test/image/export_test.js

```diff
@@ -1,40 +1,36 @@
-var fs = require('fs');
 var probeSync = require('probe-image-size/sync');
+var PNG = require('pngjs').PNG;
+var fs = require('fs');
 
 var getMockList = require('./assets/get_mock_list');
-var getRequestOpts = require('./assets/get_image_request_options');
 var getImagePaths = require('./assets/get_image_paths');
 
-// packages inside the image server docker
-var request = require('request');
-var test = require('tape');
-
 // image formats to test
 //
 // N.B. 'png' is tested in `npm run test-image, no need to duplicate here
-//
-// N.B. 'jpeg' and 'webp' lead to errors because of the image server code
-//      is looking for Plotly.Color which isn't exposed anymore
-var FORMATS = ['svg', 'pdf', 'eps'];
-
-// non-exhaustive list of mocks to test
-var DEFAULT_LIST = [
-    '0', 'geo_first', 'gl3d_z-range', 'text_export', 'layout_image', 'gl2d_12',
-    'range_slider_initial_valid'
-];
+var FORMATS = ['svg', 'jpg', 'jpeg', 'webp', 'pdf']; // exporting 'esp' requires additional dependency on CircleCI
 
-// return dimensions [in px]
-var WIDTH = 700;
-var HEIGHT = 500;
+var ONLY_FILE_SIZE = ['pdf', 'esp'];
```

Comment:
```suggestion
var ONLY_FILE_SIZE = ['pdf', 'eps'];
```
?

Reply:
oh you aren't including eps anyway.

What do you mean though by:
> // exporting 'esp' requires additional dependency on CircleCI

Reply:
Resolving now that we export and test `eps` on CircleCi: https://github.com/plotly/plotly.js/pull/5724/commits/13322c5df880f6498b1024f23f9506649dd7e0fb